tests/NCronJob.Tests/RunAtStartupJobTests.cs

```diff
@@ -37,14 +38,43 @@ public async Task UseNCronJobShouldTriggerStartupJobs()
             services.AddNCronJob(s => s.AddJob<SimpleJob>().RunAtStartup());
             services.AddSingleton(_ => storage);
         });
-        using var app = builder.Build();
+
+        using var app = BuildApp(builder);
 
         await app.UseNCronJobAsync();
 
         storage.Content.Count.ShouldBe(1);
         storage.Content[0].ShouldBe("SimpleJob");
     }
 
+    [Fact]
+    public async Task StartupJobsShouldOnlyRunOnceWhenAlsoConfiguredAsCron()
```

Comment:
Is this correct?

Imagine an EF Migration - they don't need a CRON job. They should only run once.

Reply:
That comes from a real use case.
Cache priming at startup, then regularly update it behind the scene.

Reply:
Okay - but it is still possible to execute jobs at startup without defining a cron trigger?

Reply:
Of course! All the existing tests still pass.

BTW, this doesn't change anything. It only fixes a use case.

Running the following was previously working:
```csharp
s.AddJob<SimpleJob>().RunAtStartup();
s.AddJob<SimpleJob>(jo => jo.WithCronExpression("5 * * * *"));
```

Whereas the following was not:
```csharp
s.AddJob<SimpleJob>(jo => jo.WithCronExpression("5 * * * *"));
s.AddJob<SimpleJob>().RunAtStartup();
```

This change only fixes this last case.